bitnami/gitea/1/debian-11/rootfs/opt/bitnami/scripts/gitea-env.sh

```diff
@@ -116,6 +116,7 @@ export GITEA_DATABASE_SCHEMA="${GITEA_DATABASE_SCHEMA:-}"
 export GITEA_DATABASE_CHARSET="${GITEA_DATABASE_CHARSET:-utf8}"
 export GITEA_SMTP_ENABLED="${GITEA_SMTP_ENABLED:-false}"
 export GITEA_SMTP_HOST="${GITEA_SMTP_HOST:-}"
+export GITEA_SMTP_PORT="${GITEA_SMTP_PORT:-25}"
```

Comment:
Going by [Gitea's doc](https://docs.gitea.com/next/administration/config-cheat-sheet#mailer-mailer), we should leave `GITEA_SMTP_PORT` empty by default right?

Reply:
Could you check this comment @eltorio? I think there is no need having a default value here.